app/models/quote_status.rb

```diff
@@ -0,0 +1,5 @@
+class QuoteStatus < ApplicationRecord
+  validates :name, length: { maximum: 255 }, presence: true, uniqueness: true
+  validates :order, length: { maximum: 255 }, numericality: { integer: true }, presence: true
```

Comment:
this field is a string and you are adding a `numericality` to it, it doesn't make sense

Reply:
Order will not make a math operation. So it puts it as string. And also because the form will provide this value as a string, so it will have to convert to integer to save to the database

Reply:
Okay, I will approve because I suppose that you guys already discussed the options here, but I STRONGLY suggest you to change it to use https://github.com/pluginaweek/state_machine. You guys are basically going to have to rewrite a lot of code to validate the status and to make sure it doesn't go from one state to an invalid state. This current approach is very prone errors and to be honest I'm sure that if you guys decide to add ANY kind of validations you will have to use this `order` in math operations. 

TL;DR: I'm approving because I'm only reviewing the code, but if I had to review the flow it would be reproved. 

@Heirian check this

Reply:
That was an architectural decision of @vinibr3 . They are free to decide whether they use it or not, but this kind of thing will be evaluated at the end of the project.